main.go

```diff
@@ -46,16 +46,16 @@ func main() {
 		// there is no need for second parameter anymore. The purpose is overwriting rootCmd as it is at the beginning.
 		rootCmd, _ = rootcmd.New(&cfg.Hazelcast)
 		err = RunCmd(ctx, rootCmd)
-		ExitOnError(err)
+		ExitOnError(err, cfg.Logger)
 	}
 	return
 }
 
-func ExitOnError(err error) {
+func ExitOnError(err error, logger *log.Logger) {
 	if err == nil {
 		return
 	}
 	errStr := HandleError(err)
-	fmt.Println(errStr)
+	logger.Println(errStr)
```

Comment:
why does `[Hazelcast Go Client]` prefix does not apply to this log (I tested it)? just curious 🤔

Reply:
Since this is a CLC-specific log, I thought we can differentiate those log sources with such a prefix. We use different logger instances for CLC and Go-client. The one we pass to Go-client has this prefix set.